core/src/sumeragi/message.rs

```diff
@@ -110,6 +110,7 @@ impl Message {
     #[log(skip(self, sumeragi))]
 
     pub fn handle<F: FaultInjection>(self, sumeragi: &mut SumeragiWithFault<F>) -> Result<()> {
+        println!("Hello there you");
```

Comment:
I'm using println because logging output is disable for some reason in the test `integration::unstable_network::unstable_network_4_peers_1_fault`. Of course they will be removed or replaced by logging before we merge into iroha2-dev. But for now they stay.